engines/ags/plugins/ags_waves/sound.cpp

```diff
@@ -47,7 +47,7 @@ void AGSWaves::SFX_Play(ScriptMethodParams &params) {
 	_mixer->stopHandle(effect._soundHandle);
 
 	Common::FSNode fsNode = ::AGS::g_vm->getGameFolder().getChild(
-		"sounds").getChild(Common::String::format("sound%d.sfx", sfxNum));
+		"Sounds").getChild(Common::String::format("sound%d.sfx", sfxNum));
```

Comment:
Is this uppercase in all games that use this plugin?
It would be best to check for the existence of either the "sounds" or the "Sounds" folder, and use the one that is found

Reply:
Our filesystem code is supposed to be case insensitive so this should not be a problem.

Reply:
> Is this uppercase in all games that use this plugin?

I think only Strangeland uses this plugin, but the additional check wouldn't hurt

> Our filesystem code is supposed to be case insensitive so this should not be a problem.

I thought the same... a fix in COMMON would be preferable but maybe it could impact other engines such as Director?

Reply:
> Our filesystem code is supposed to be case insensitive so this should not be a problem.
> 
> EDIT: After looking at the common code, it doesn't do anything to handle this problem, so I am not sure anymore where the fix should be (AGS or COMMON)

That was my expectation when I started look into common filesystem code. But I couldn't find anything to handle case-sensitivity at global level yet I've found commits to fix stuff engine level like that one: https://github.com/scummvm/scummvm/commit/7c23eae102

Reply:
This must be fixed in common

Reply:
It's pretty common for Linux users to lowercase game files when they are using a native engine. Sometimes that causes problems, but it mostly solves them, so it's a first go for anyone that doesn't have a case insensitive filesystem at hand.